training/templates/training/stats.html

```diff
@@ -2,81 +2,87 @@
 {% load humanize %}
 {% load static %}
 
+
+{% block head %}
+<style media="all">
+  .card-title {
+    text-align: center;
+    font-size: 4rem;
+  }
+</style>
+{% endblock %}
+
 {% block content %}
-<h2>Overview</h2>
-<div class="row">
-  <div class="col-sm-3">
+
+<div class="container">
+  <h2>Overview</h2>
+  <p>Data is cumulative since {{ earliest }}</p>
+
+  <br><br>
+
+  <div class="card-group">
     <div class="card">
       <h5 class="card-header">Today</h5>
       <div class="card-body">
         <h1 class="card-title">{{ current_trainings }} <i class="fas fa-chalkboard-teacher"></i></h1>
         <p class="card-text">Training events happening today</p>
       </div>
     </div>
-  </div>
-  <div class="col-sm-3">
     <div class="card">
       <h5 class="card-header">Overall</h5>
       <div class="card-body">
-        <h1 class="card-title">{{ approved }} <i class="fas fa-calendar-alt"></i> </i></h1>
+        <h1 class="card-title">{{ approved }} <i class="fas fa-calendar-alt"></i> </h1>
         <p class="card-text">Events since {{ earliest }}, with {{ waiting }} new training requests waiting</p>
       </div>
     </div>
-  </div>
-  <div class="col-sm-3">
     <div class="card">
       <h5 class="card-header">Overall</h5>
       <div class="card-body">
         <h1 class="card-title">{{ students }} <i class="fas fa-user-graduate"></i></h1>
         <p class="card-text">students taught over the lifetime of the TIaaS service</p>
       </div>
     </div>
-  </div>
-  <div class="col-sm-3">
     <div class="card">
       <h5 class="card-header">Overall</h5>
       <div class="card-body">
-		  <h1 class="card-title">{{ days }} <i class="fas fa-laptop"></i></h1>
-		  <p class="card-text">Days of compute resources provided various training events.</p>
+        <h1 class="card-title">{{ days }} Days</h1>
```

Comment:
```suggestion
        <h1 class="card-title">{{ days }} <i class="fas fa-laptop"></i></h1>
```
guessing this was a rebase accident

Reply:
Done